src/utils/discord/logger.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const padZeros = (value, length = 2) => {
+  return value ? value.toString().padStart(length, '0') : '';
+};
+
+const formatDate = date => {
+  const month = padZeros(date.getMonth() + 1);
+  const day = padZeros(date.getDate());
+  const year = date.getFullYear();
+  return [year, month, day].join('-');
+};
+
+const formatTime = date => {
+  const hours = padZeros(date.getHours());
+  const minutes = padZeros(date.getMinutes());
+  const seconds = padZeros(date.getSeconds());
+  return [hours, minutes, seconds].join(':');
+};
+
+const log = ({ userName, userId, action, message, stack }) => {
+  const dateNow = new Date();
+  const date = formatDate(dateNow);
+  const time = formatTime(dateNow);
```

Comment:
you can do the formatting with date-fns easier, ie.
`
import { format } from 'date-fns';
const date = format(new Date, 'yyyy-MM-dd');
`
see https://date-fns.org/v2.16.1/docs/format

Reply:
Ok nice, I'll use this and remove the need for padZeros with it :)